### ISSUE TITLE
Solution: 02_Ten/04_loss_..., the last part with  tf.nn.sparse_softmax_cross_entropy_with_logits

### ISSUE THREAD

**Issue by leigaoyi** [nfmcclure/tensorflow_cookbook#30]
I'm not quite understand how this function work, but in my computer. Do the following changes can make code works.Otherwise, errors are given.
Tensorflow 1.0.0

**sparse_xentropy = tf.nn.sparse_softmax_cross_entropy_with_logits(logits=unscaled_logits,labels= sparse_target_dist)
print(sess.run(sparse_xentropy))**

**Comment by nfmcclure** [comment]
It should be fixed now.

Needless to say the upgrade to TF v 1+ has been pretty harsh across the board.

Thanks for this notice.  Let me know if you have any more problems!
